scripts/lib/constants.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+USER_TYPES = {
+        'basic': 1,
+        'pro': 2,
+        'corp': 3,
+        }
```

Comment:
Minor formatting nit… either line things up with the brackets:

```suggestion
USER_TYPES = {'basic': 1,
              'pro': 2,
              'corp': 3 }
```

Or use standard 4-space indentation:

```suggestion
USER_TYPES = {
    'basic': 1,
    'pro': 2,
    'corp': 3,
}
```